WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -497,7 +497,11 @@ extension OrdersViewController: UITableViewDelegate {
             assertionFailure("Expected OrderDetailsViewController to be instantiated")
             return
         }
-        orderDetailsVC.viewModel = viewModel.detailsViewModel(at: indexPath)
+        let orderDetailsViewModel = viewModel.detailsViewModel(at: indexPath)
+        orderDetailsVC.viewModel = orderDetailsViewModel
+
+        ServiceLocator.analytics.track(.orderOpen, withProperties: ["id": orderDetailsViewModel.order.orderID,
+        "status": orderDetailsViewModel.order.statusKey])
```

Comment:
I'm not sure how I should feel that Hound didn't complain about this indentation but complains about extra spaces. 🙄

Reply:
In this case, I believe that the indentation is ok, and does not exceed 160 characters 👍